### ISSUE TITLE
Is it practical to test "unfound" Statements due to voided?

### ISSUE THREAD

**Issue by andyjohnson** [TryxAPI/lrs-conformance-test-suite#25]
It seems that an LRS may be able to reject a voiding Statement because it doesn't have a Statement Id in the system.  The tests following that may have to adjust.  In other words, this test is only a SHOULD (or SHOULD NOT reject, if you will), even for 1.0.3.

**Comment by DavidTPate** [comment]
@andyjohnson The part where it gets weird is that the specification states:

> A Statement Reference MUST set the "id" property to the UUID of a Statement. There is no requirement for the LRS to validate that the UUID matches a Statement that exists.

When describing a [StatementRef](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#stmtref), while there is no requirement that I could find around [Voided](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#43-voided) needing any additional requirements.

**Comment by andyjohnson** [comment]
Hey @DavidTPate thanks!  You are correct.  We changed the closest language in 1.0.1/1.0.2 

"Upon receiving a Statement that voids another, the LRS SHOULD return a descriptive error if the target Statement cannot be found."

to

"Upon receiving a Statement that voids another, the LRS SHOULD NOT* reject the request on the grounds of the Object of that voiding Statement not being present."

with the reasoning that the only way an error can be returned realistically is with a rejection.  We don't have any sort of warning in place, even though that was the intent.  We effectively created gray area, which we now seek to undo. 

We decided on the call that it would be a good idea to have some of these SHOULD* and SHOULD NOT* behaviors as flags or warnings in the Test Suite.
